### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by Abhiram716**
# Desktop version
Screenshot from 2023-07-15 15-15-11

# Mobile Version
Screenshot from 2023-07-15 15-17-48

This page looks awkwardly blank to the right side. looking for any suggestions.

### CODE REVIEW

File: src/theme/breakpoints.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const breakpoints = {
+  values: {
+    xs:0,
```
Comment (aadilmughal786) [inline src/theme/breakpoints.js:3]: Will be like `xs: 0,`

File: src/pages/index.jsx [hunk 2]
```diff
@@ -3,12 +3,17 @@ import Container from "@mui/material/Container";
 import CssBaseline from "@mui/material/CssBaseline";
 import React from "react";
 import theme from "../theme/theme";
+import HomePage from "../components/home-page";
```
Comment (aadilmughal786) [inline src/pages/index.jsx:7]: We should separate local and global import for better readability

**Review by aadilmughal786 — APPROVED** [review]
looks good to me bro

**Comment by Abhiram716** [comment]
This page is not done yet. I am still looking for improvements in page design

**Pull request closed without merging**